aether/sdk/management/commands/cdn_publish.py

```diff
@@ -91,11 +91,12 @@ def handle(self, *args, **options):
         self.__publish(webpack_dir, storage_path)
 
     def __publish(self, local_dir, storage_path):
+        static_storage_class = get_storage_class(settings.STATICFILES_STORAGE)()
```

Comment:
if you move this line outside the class, I think the tests will pass :crossed_fingers: